pyflakes/checker.py

```diff
@@ -1292,13 +1317,21 @@ def TUPLE(self, node):
 
     def IMPORT(self, node):
         for alias in node.names:
+            name = alias.name
             if '.' in alias.name and not alias.asname:
-                importation = SubmoduleImportation(alias.name, node)
+                importation = SubmoduleImportation(name, node)
             else:
                 name = alias.asname or alias.name
-                importation = Importation(name, node, alias.name)
+                importation = Importation(name, node, name)
```

Comment:
Are you sure about this? The 3rd arg of `Importation` originally always pass `alias.name` (the fullname of import) and now it could have value of `alias.asname`. The meaning of code changes.

Reply:
oh, sorry. it's my mistake